.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - "3.1"
+          - "3.2"
+          - "3.3"
+        gemfile:
+          - gemfiles/rails_7_0.gemfile
+          - gemfiles/rails_7_1.gemfile
+          - gemfiles/rails_main.gemfile
```

Comment:
This only tests with Rails >= 7.0 and Ruby >= 3.1. I can add entries for Rails 6.0 and 6.1, and older versions of Ruby, but first I wanted to confirm which versions we want to support.